apps/services/temporary-voter-registry-api/src/app/modules/voterRegistry/voterRegistry.controller.ts

```diff
@@ -0,0 +1,22 @@
+import { Controller, Get, NotFoundException, Query } from '@nestjs/common'
+import { FindOneDto } from './dto/findOne.dto'
+import { VoterRegistry } from './voterRegistry.model'
+import { VoterRegistryService } from './voterRegistry.service'
+
+@Controller('voter-registry')
+export class VoterRegistryController {
+  constructor(private readonly voterRegistryService: VoterRegistryService) {}
+
+  @Get()
```

Comment:
Authorization?

Reply:
I don't think the current timeframe allows for implementation of OAuth2, but I see where you are coming from.
Our goal is to present a working system by thursday.
After that deadline the user flows and functionality need to be tested probably accompanied by some iterations.
I'm confident we can implement auth into this service during that timeframe